.NET/Microsoft.Recognizers.Text.NumberWithUnit/French/Parsers/FrenchNumberWithUnitParserConfiguration.cs

```diff
@@ -21,7 +21,7 @@ public FrenchNumberWithUnitParserConfiguration(CultureInfo ci)
             this.InternalNumberExtractor = NumberExtractor.GetInstance(numConfig);
             this.InternalNumberParser = AgnosticNumberParserFactory.GetParser(AgnosticNumberParserType.Number,
                                                                               new FrenchNumberParserConfiguration(numConfig));
-            this.ConnectorToken = NumbersWithUnitDefinitions.ConnectorToken;
+            this.ConnectorToken = NumbersWithUnitDefinitions.ConnectorTokenWithSpace;
```

Comment:
As this is a French-only wokaround. I wonder if it wouldn't be simpler to just append a space to ConnectorToken in this specific config and have a code comment explaining the workaround.
Where exactly doesn't problem happen of mixing the connector and the unit term?

Reply:
yes, it would be the same, I will make the change.

When the extracted text is processed by the Parser, it is split at the found numbers in order to extract the units and the connector ends up attached to the following unit since there is no number between them (in the for loop at line 71 of NumberWithUnitParser).
Then later the connector is stripped from the unit if the unit string is found to start with the connector (line 111).

Reply:
moved space to French config file